docs/shared/migration/adding-to-existing-project.md

```diff
@@ -23,120 +18,225 @@ You can make these scripts faster by leveraging Nx's caching capabilities. For e
-
-{% tabs %}
-{% tab label="Nx >= 17" %}
+This will set up Nx for you - updating the `package.json` file and creating a new `nx.json` file with Nx configuration based on your answers during the set up process. The set up process will suggest installing Nx plugins that might be useful based on your existing repository. The example below is using the `@nx/eslint` and `@nx/next` plugins to run ESLint and Next tasks with Nx:
 
 ```json {% fileName="nx.json" %}
 {
-  "targetDefaults": {
-    "build": {
-      "cache": true
+  "plugins": [
+    {
+      "plugin": "@nx/eslint/plugin",
+      "options": {
+        "targetName": "lint"
+      }
     },
-    "lint": {
-      "cache": true
+    {
+      "plugin": "@nx/next/plugin",
+      "options": {
+        "buildTargetName": "build",
+        "devTargetName": "dev",
+        "startTargetName": "start"
+      }
     }
-  }
+  ]
 }
 ```
 
-{% /tab %}
-{% tab label="Nx < 17" %}
+When Nx updates your `package.json` scripts, it looks for scripts that can be non-intrusively replaced with an Nx command that has caching automatically enabled. The `package.json` defined above would be updated to look like this:
 
-```json {% fileName="nx.json" %}
+```json {% fileName="package.json" %}
 {
-  "tasksRunnerOptions": {
-    "default": {
-      "runner": "nx/tasks-runners/default",
-      "options": {
-        "cacheableOperations": ["build", "lint"]
-      }
-    }
+  "name": "my-workspace",
+  ...
+  "scripts": {
+    "build": "nx build",
+    "lint": "nx lint",
+    "test": "node ./run-tests.js"
+  },
+  ...
+  "nx": {
+    "includedScripts": []
   }
 }
 ```
 
-{% /tab %}
-{% /tabs %}
+The `@nx/next` plugin can run `next build` for you and set up caching correctly, so it replaces `next build` with `nx build`. Similarly, `@nx/eslint` can set up caching for `eslint ./src`. When you run `npm run build` or `npm run lint` multiple times, you'll see that caching is enabled. You can also call Nx directly from the terminal with `nx build` or `nx lint`.
 
-## Wrapping Cacheable Scripts
+The `test` script was not recognized by any Nx plugin, so it was left as is.
 
-Nx also automatically wraps your cacheable scripts with the `nx exec` command. The main advantage here is that you can still keep using `npm start` or `npm run build` (or other package manager's alternatives) as you're accustomed to. But still get the benefits of making those operations cacheble.
+## Inferred Tasks
 
-Here's an example of a `build` and `lint` script being wrapped by Nx:
+You may have noticed that `@nx/next` provides `dev` and `start` tasks in addition to the `build` task. Those tasks were created by the `@nx/next` plugin from your existing Next configuration. To view all available tasks, open the Project Details view with Nx Console or use the terminal to launch the project details in a browser window.
```

Comment:
We should refer to Next as "Next.js" since that's how it is referenced in official docs and other content.